benchexec/tools/fdse.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for FDSE.
+    https://github.com/zbchen/FDSE
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("fdse")
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
+        )
+
+    def version(self, executable):
+        return "fdse-testcomp"
+
+    def cmdline(self, executable, options, task, rlimits):
+        new_option = ["--testcomp"]
```

Comment:
Is this an option that is always required to run your tool together with this tool-info module, or is this a configuration option that would not be necessary in other circumstances, e.g., outside of Test-Comp? Then please leave it out and put it in the benchmark definition for Test-Comp.

BenchExec is a general benchmarking framework and its use is not restricted to Test-Comp, so we want to keep the code general here.

Reply:
Good idea !